tickets_created.tsv

```diff
@@ -0,0 +1,20 @@
+Trac URL	GitHub URL	Comment
```

Comment:
why is this added to the repo?

Is ok to have this file generated, but I think that it should stay outside of the repo.